docs/slides/content/social-contracts.md

```diff
@@ -51,38 +51,38 @@ TL500 <!-- {.element: class="title-color"} -->
 
 The Social Contract is a simple, yet highly effective manner to enable team autonomy and self accountability for engagements.<!--{.element: style="font-size: ; font-weight: 400!important;"} -->
 
-A Social Contract codifies behavior and expectations of the team between its members.<!--{.element: style="font-size: ; font-weight: 400!important;"} -->
+A Social Contract codifies behavior and expectations for the team members.<!--{.element: style="font-size: ; font-weight: 400!important;"} -->
 
 
 
-#### Effective Social Contracts are
+#### Effective Social Contracts
 - Publicly displayed
 - Revisited regularly and amended if required
 - Binding on *all* parties
 - Signatories hold each other accountable
 
 
 
-#### _Why use it?_
```

Comment:
The format of the slides for all practices follows this approach of conversational English. For all the slides you should see this approach of `What is it?` `Why use it?` and `How do we do it?`

Reply:
Ok, I don't really agree with using "it" as the subject, especially in a heading, but I will revert those changes and ignore them in future if that's the decision the team has made.

Reply:
Ok, I've reverted these changes in a new commit. :)